lib/register.js

```diff
@@ -1,26 +1,4 @@
 'use strict';
```

Comment:
Not sure tbh, but maybe you are right.. It might be nice to do a major release as well to minimize the effects of any regressions since there are a lot of changes.

So let's remove this file also as we don't care about backwards compatibility!